Open Judge System/OJS.Services/Data/SubmissionsForProcessing/SubmissionsForProcessingDataService.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace OJS.Services.Data.SubmissionsForProcessing
+{
+    using System.Linq;
+
+    using EntityFramework.Extensions;
+
+    using OJS.Data.Repositories.Contracts;
+
+    public class SubmissionsForProcessingDataService : ISubmissionsForProcessingDataService
```

Comment:
Since we are starting to implement a Service layer pattern it would be a good idea to extract all the non-generic methods from ISubmissionsForProcessingRepository into  ISubmissionsForProcessingDataService  alongside with their implementations.

Reply:
For further improvement and the removal of the empty ISubmissionsForProcessingRepository  and SubmissionsForProcessingRepository there is another issue https://github.com/SoftUni-Internal/suls-issues/issues/3247